live/README.md

```diff
@@ -0,0 +1,240 @@
+# Live ISO
+
+<!-- omit from toc -->
+## Table of Content
+
+- [Live ISO](#live-iso)
+  - [Layout](#layout)
+  - [Building the Sources](#building-the-sources)
+  - [Building the ISO Image](#building-the-iso-image)
+    - [Build Options](#build-options)
+  - [Image Definition](#image-definition)
+    - [Kiwi Files](#kiwi-files)
+  - [Image Configuration](#image-configuration)
+    - [SSH Server](#ssh-server)
+    - [Autologin](#autologin)
+    - [Firefox Profile](#firefox-profile)
+    - [Dracut menu](#dracut-menu)
+    - [Avahi/mDNS](#avahimdns)
+      - [The Default Hostname](#the-default-hostname)
+      - [Service Advertisement](#service-advertisement)
+    - [The Default Cockpit/Agama TCP Port](#the-default-cockpitagama-tcp-port)
+    - [Autoinstallation Support](#autoinstallation-support)
+    - [Firmware Cleanup](#firmware-cleanup)
+
+---
+
+## Layout
+
+This directory contains a set of files that are used to build the Agama Live ISO
+image.
+
+- [src](src) subdirectory contains all source files which are copied unmodified
+  to the OBS project
+- [root](root) subdirectory contains files which are added to the Live ISO root
+  system (inside the squashfs image)
+- [root-ALP-PXE](root-ALP-PXE) subdirectory contains specific files for the ALP
```

Comment:
I guess we will need to drop this ALP reference at some point. But it is out of scope of this PR.